app/grandchallenge/challenges/models.py

```diff
@@ -1229,36 +1229,60 @@ def base_cost_euros(self):
                 exempt_from_base_costs=True, members_group__user=self.creator
             ).exists()
         ):
-            return 0
+            return settings.CHALLENGE_MINIMAL_COMPUTE_AND_STORAGE_IN_EURO
```

Comment:
These changes mean the property name is no longer correct.

Reply:
Or maybe it is better to create a new property as now this property does two things. The "base_costs" calculation is still valid in the new case.

Reply:
What would you call the second property? `minimal_challenge_cost`?

Reply:
Yes sounds fine.